app/src/main/res/values/strings.xml

```diff
@@ -529,7 +529,7 @@
     <string name="auto">Auto</string>
     <string name="switch_view">Switch View</string>
     <!-- App update notification -->
-    <string name="app_update_notification_content_title">NewPipe Update Available!</string>
+    <string name="app_update_notification_content_title">There is a fresh NewPipe version available!</string>
```

Comment:
This is a notification title which should not be too long. Otherwise last part of it will be replaced by  "..."

Reply:
The shortest alternative is "New NewPipe available" (?)

Reply:
imo "NewPipe Update Available!" sounds fine. This change should be reverted.

Reply:
@cool-student The problem is with update/upgrade. In Debian that is an upgrade, but some are calling it an update.

Reply:
There is no way to determine if it's an Update or Upgrade since one would need to know what changed each version.

Reply:
That is yet another way to interpret update/upgrade, when "new version" does away with the confusion.

Reply:
What about "New NewPipe version available!"

Reply:
I would argue that Update has enough expressive power especially when you consider that most people aren't power users and these casual users will understand what an Update is. There is no such difference of Update or Upgrade for the typical user.

Reply:
So since I found a way to express it to all users, that some of them are not affected is a good reason to trip the others up? It is very plausible you could update something else in NewPipe, like the video lists, subscriptions, etc.

Reply:
Please keep the old one, because this is text for a notification and there is only limited space for notification headings.